core/validation/utils.py

```diff
@@ -12,3 +19,48 @@ def remove_duplicate_indexes(df: pd.DataFrame):
     :return: the DataFrame without duplicate indexes
     """
     return df[~df.index.duplicated(keep="first")]
+
+
+def get_cell_indexes_for_outcomes(failed_row: pd.Series, index: int) -> str:
```

Comment:
Do we need to pass the series and the index? We can obtain the index from the series I think?

Reply:
When we iterate through the DataFrame, we iterate through the index and row as two separate values.

```
for index, row in workbook[sheet_name].iterrows():
```

The row does not contain the index by default. The index of the Series is the name of the columns. We could bodge 'original_index' in as an extra column but this feels simpler

Reply:
Although come to think of it, we have accessed the index before with Series.name. Let me investigate if that will work here

Reply:
Ah ok, we can access the index with the .name attribute, but it wouldn't work with unit tests unless .name is explicitly set after instantiating the Series

Reply:
You can set name at the same time as instantiating the Series e.g.

```
pd.Series([1, 2, 3], dtype=np.int64, name='Numbers')
```